goiscsi_iscsi.go

```diff
@@ -37,6 +39,8 @@ const (
 	// ISCSINoObjsFoundExitCode exit code indicates that no records/targets/sessions/portals
 	// found to execute operation on
 	iSCSINoObjsFoundExitCode = 21
+	// Timeout for iscsiadm command to execute
```

Comment:
does this affect other driver who are using goiscsi ? using this command shouldn't change the usage of lib with other platform drivers, I would request other driver test results before we merge this PR.

Reply:
Thank you @prablr79  , PowerMax and CSI-Unity team has confirmed that this one is not impacting the driver functionality. We have tested it for PowerStore too.
We have approval from both team to go ahead with this PR.